netmaster/mastercfg/policyState.go

```diff
@@ -43,6 +43,7 @@ type RuleMap struct {
 type EpgPolicy struct {
 	core.CommonState
 	EpgPolicyKey    string              // Key for this epg policy
+	EpCount         int                 // Endpoint Count for this policy
```

Comment:
Lets remove this attribute too since its not being used..
Its hard to keep track of ep count at epgPolicy level since epgPolicy does not change as new endpoints are created/destroyed within an EPG